### DISCUSSION TITLE
Total purchase price is missing the quotation module

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15578] [Feature improvement]
hi,

When we add the products to the quotation module, it always displays the purchase price per qty. When we increase the qty of the products, we wish to calculate the total purchase price for the selected products. It automatically calculates the margin percentage based on the total purchase cost. But it doesnt display the total purchase cost column.

How to display the total purchase cost in the quotation module or how to create one custom field having  the values of Purchase cost per unit x qty in the quotation module. ( when we try to add the workflow for this calculation, we are unable to find the fields in the products table). 

Please provide the solution for the same.

Thanks in advance.

**Comment by mariuszkrzaczkowski** [comment]
attach a screenshot

**Reply by Ilamparithi-Tamizhanangu** [reply]
hi,

Please find the below attachment.

![image](https://user-images.githubusercontent.com/96602176/147423159-07d498d3-fc66-4d34-a05f-db5773d57f9e.png)

The unit purchase price is displayed as 900. qty = 2. The total purchase price is 1800.

total purchase cost is used in the margin % calculation.

Margin % = (Total selling cost - total purchase cost) / Total purchase cost x 100.

We required the total purchase price while generating the quotations.

Could you please guide us to enable this field. 

Thanks in advance. 

with regards,
Rajasekaran.

**Reply by Ilamparithi-Tamizhanangu** [reply]
Hi @mariuszkrzaczkowski ,

Do we have any option to display the total purchase cost in the existing CRM funcationalities or does it required any development ?

kindly confirm the possibilities to add it. 

Thanks & Regards,
Rajasekaran.
